RxBlocking/BlockingObservable+Operators.swift

```diff
@@ -101,9 +101,9 @@ extension BlockingObservable {
 }
 
 extension BlockingObservable {
-    private func materializeResult(max: Int? = nil, predicate: @escaping (Element) throws -> Bool = { _ in true }) -> MaterializedSequenceResult<Element> {
-        var elements = [Element]()
-        var error: Swift.Error?
+    private func materializeResult(max: Int? = nil, predicate: @escaping @Sendable (Element) throws -> Bool = { _ in true }) -> MaterializedSequenceResult<Element> {
+        nonisolated(unsafe) var elements = [Element]()
```

Comment:
Because `elements` is accessed both in the function body and in the `subscribe` closure. The `subscribe` closure is not necessarily executed in sync or on the same thread as `materializeResult`